docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: "3"
+
+services:
+  app:
+    restart: always
+    build: .
+    ports:
+      - "3000"
+    depends_on:
+      - mongo
+      - redis-server
+
+  mongo:
+    container_name: mongo
```

Comment:
This is not necessary, you can let it take the container name, or else while composing if you want to increase the mongo instances it might throw an exception, not sure what will be the behaviour.